features/accessibility.feature

```diff
@@ -0,0 +1,39 @@
+@javascript
+Feature: Accessibility of pages
+  As a customer I want Pension Wise pages to be fully accessible
+  so I can navigate the website regardless of my disability
+
+  Scenario: Homepage of the site is accessible
+    When I visit the homepage
+    Then the page should be accessible
+
+  Scenario Outline: Typical pages of the site are accessible
+    When I visit <Slug> page
+    Then the page should be accessible
+
+    Examples:
+      | Slug                     |
+      | facebook-landing         |
+      | leave-pot-untouched      |
+      | tax                      |
+      | summary-document/new     |
+      | feedback/new             |
+
+  @vcr
+  Scenario: Location page is accessible
+    Given I have searched for appointment locations near to a valid postcode
+    When I drill down into a specific search result
+    Then the page should be accessible
+
+  Scenario: Appointment summary page is not accessible until errors have been fixed
+    When I visit the digital appointment summary generator
+    And I select 55 or over as my age range
```

Comment:
I'm kinda questioning the actual feature specs to be honest - as i'm piecing together some steps in order to get to the page I want to test for accessibility.

Maybe I need something a lot simpler as a feature spec, which then follows some steps in the background.